src/authn/ccloudProvider.test.ts

```diff
@@ -439,6 +440,11 @@ describe("CCloud auth flow", () => {
     await ccloud.deleteCCloudConnection();
   });
 
+  afterEach(async () => {
+    // force sign-out so we don't have a lingering CCloud connection for other tests
+    await ccloud.deleteCCloudConnection();
+  });
```

Comment:
@jlrobins I'm adding this in here based on previous discussions around cross-test state contamination